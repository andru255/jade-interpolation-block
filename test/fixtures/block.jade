!=
  this.headTitle('My awesome title ' + this.translate('foo bar'))
       .setSeparator('-')
       .setAutoEscape(false)

!=
  this.headTitle('My awesome title ' + this.translate('foo bar'))
      .setSeparator('-')
      .setAutoEscape(false)