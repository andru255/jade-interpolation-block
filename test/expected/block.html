<title>Myawesometitle translated:foobar separatorwith:-</title>
<title>Myawesometitle translated:foobar separatorwith:-</title>
